platforms/ios/src/TangramMap/TGHelpers.mm

```diff
@@ -36,10 +36,14 @@ + (TGError)convertTGErrorTypeFrom:(Tangram::Error)error
             return TGErrorSceneUpdatePathNotFound;
         case Tangram::Error::scene_update_value_yaml_syntax_error:
             return TGErrorSceneUpdateValueYAMLSyntaxError;
+        case Tangram::Error::no_valid_scene:
+            return TGErrorNoValidScene;
+        case Tangram::Error::none:
+            return TGErrorNone;
     }
 }
 
-+ (NSError *)errorFromSceneUpdateError:(Tangram::SceneUpdateError)updateError
++ (NSError *)errorFromSceneError:(Tangram::SceneError)updateError
```

Comment:
Updated here: d5a7d16.

I think what error info I added make sense for current context. Will need to update this when we handle more error states in Tangram. Feel free to update the language if need be.